include/glow/Graph/Nodes.h

```diff
@@ -108,21 +108,35 @@ class Constant : public Storage {
   bool verify() const;
 };
 
+uint32_t getNewPlaceholderId();
+
 /// Placeholder nodes are unbound-storage. The content tensors are attached to
 /// this node at runtime. Placeholders are used as inputs and output nodes to
 /// the network.
 class Placeholder : public Storage {
   /// Specifies if the placeholder is trainable.
   bool isTrainable_;
+  // TODO: comment
```

Comment:
I think that this should go in the module. The module will be able to unique and translate between the ID and the pointer.